config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests", to: "guests#show"
```

Comment:
For a show route, you will need the ID from the path to determine what guest you want to show. The verb, controller, and action are all correct, but the path should be: `"/guests/:id"`. This way we get to use `params[:id]` in the controller to find the guest we want to show.